getting-started/exploring-labs.md

```diff
@@ -0,0 +1,93 @@
+# Setup [](setup)
+Now that you’ve been invited to Timescale Labs, you’re ready to work 
+with some data. The first thing to do is to create a new database. 
+Timescale Labs is a PostgreSQL database with TimescaleDB extensions 
+already installed. Timescale Labs automatically provisions and manages 
+a TimescaleDB instance in Amazon Web Services.
+
+>:WARNING: Timescale Labs is in private alpha. [Request an invite][invite-request] . We will keep this document updated as we add features, but the screenshots and instructions may be temporarily out of date.
+
+## Sign in to Timescale Labs [](signin)
+Sign in to Timescale Labs and you’ll see that a `timescale_demo` 
+service is automatically created for you. Click on the service 
+and you’ll see a screen similar to this:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/labs_config_screen.png" alt="timescale labs config screen"/>
```

Comment:
The image was located on `assets.timescale.com` so src wasn't working, but I just uploaded it to iobeam.com as well